tests/e2e/flowtest/test_cephfilesystem_creation.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.helpers.helpers import create_resource, validate_cephfilesystem
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility import templating
+from ocs_ci.ocs.ocp import OCP
+
+
+logger = logging.getLogger(__name__)
+cephfs_data = templating.load_yaml(constants.CEPHFILESYSTEM_YAML)
```

Comment:
Why not use https://github.com/red-hat-storage/ocs-ci/blob/89463d2e1e900907791ade33af943a4410bf68e9/ocs_ci/helpers/helpers.py#L441
I think if you give the pool name `test-ceph-fs` it will work.

Reply:
@shyRozen It has a comment that "** This method should not be used anymore **" thus I haven't used it.